compiler/src/dmd/expressionsem.d

```diff
@@ -7813,6 +7813,18 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             if (exp.to == Type.terror)
                 return setError();
         }
+        if (global.compileEnv.obsolete)
+        if (auto toc = exp.to.isClassHandle())
```

Comment:
This style is kind of weird. Is this intended? (if's having the same indentation).